compute/instances/api-cli/using-placement-groups.mdx

```diff
@@ -27,15 +27,15 @@ image: /assets/images/docs/scaleway.png
 ---
 
 
-Placement Groups allow you to group instances into groups.
+placement groups allow you to group instances into groups.
```

Comment:
Start of a sentence

Reply:
```suggestion
Placement groups allow you to group instances into groups.
```

Reply:
I think for this and some of the other things I commented on, you resolved the conversation without commiting the suggestions? It looks like this (and some other things) are still in lower case in the file.